dependencies.yaml

```diff
@@ -720,7 +720,7 @@ dependencies:
         matrices:
           - matrix: {dependencies: "oldest"}
             packages:
-              - numba==0.57.*
+              - *numba-cuda-dep
```

Comment:
Note that I changed this so that we now require numba-cuda even to test the oldest supported dependency. If at some point we choose to support a range of versions then we should remove the alias here and insert a real version.